lib/ansible/modules/cloud/amazon/cloudfront_info.py

```diff
@@ -504,7 +504,7 @@ def get_distribution_id_from_domain_name(self, domain_name):
         try:
             distribution_id = ""
             distributions = self.list_distributions(False)
-            distributions.update(self.list_streaming_distributions(False))
+            distributions += self.list_streaming_distributions(False)
```

Comment:
``` distributions.extend(self.list_streaming_distributions(False))```
```+=``` may be fast , `extend` bring clarity.

Reply:
but both work for lists only, right? 
The question remains how I managed to run into the initial error described here https://github.com/ansible/ansible/pull/66695#issue-365945592 as I can't reproduce it anymore. The problem was that somehow both elements were dicts in that case. But now the `update` won't work anymore as in fact both elements are lists now, but I am still on python 3.7.

Reply:
```list_streaming_distributions(self, keyed=True)``` function seems to be returning dict. Shouldn't returning `list` be the erorr.

Reply:
I am not sure at this point. I can do more testing if you want. All I figured out yesterday was that the cloudfron_info was broken for me so I reverted my initial "fix" which resolved the issue.

Reply:
If you can share how are you testing it, I can also give it a try.

Reply:
I use

```yaml
cloudfront_info:
  domain_name_alias: 'mydomain.alias.com'
  distribution_config: yes
  region: us-east-1
```
which doesn't work with `distributions.update()` fix being present.